AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -69,17 +86,17 @@ CAmSocketHandler::CAmSocketHandler() :
     short event = 0;
     sh_pollHandle_t handle;
     event |= POLLIN;
-    if (addFDPoll(mPipe[0], event, NULL,
-                [](const pollfd, const sh_pollHandle_t, void*){},
-                [](const sh_pollHandle_t, void*) { return (false); },
-            NULL, NULL, handle) != E_OK)
-    {
+    if (addFDPoll(mPipe[0], event, NULL, [](const pollfd pollfd, const sh_pollHandle_t, void*)
```

Comment:
There's a really small code change here, in the middle of reformatting! 
Was it intended?  How will a reviewer know...   It should not change any behavior, but still it is there...
It is extremely easy to miss and to just assume that this is only reformatting here!